src/components/SearchBar.jsx

```diff
@@ -95,7 +96,8 @@ class SearchBar extends Component {
           }
         })
 
-        window.addEventListener('message', this.onMessageFromSource(this.sources))
+        var debounced = debounce(this.onMessageFromSource(this.sources), 250, { 'maxWait': 1000 })
```

Comment:
Shouldn't we use `const` instead of `var` here?

Reply:
han, pris en flag de copy/paste development ! Déso, je change ça, évidemment qu'on reste sur du `const`.